test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Source/AddTagHelperDirective.cshtml

```diff
@@ -1 +1 @@
-﻿@addTagHelper "**??**??*, nice"
\ No newline at end of file
+﻿@addTagHelper something, nice
```

Comment:
Why did this previously work? Didn't think we'd supported more than "something*" for a while.